rclcpp/include/rclcpp/callback_group.hpp

```diff
@@ -55,19 +56,19 @@ class CallbackGroup
 
   RCLCPP_PUBLIC
   const std::vector<rclcpp::subscription::SubscriptionBase::WeakPtr> &
-  get_subscription_ptrs() const;
+  get_subscription_ptrs();
```

Comment:
8b6a417f8f405dbff4f0132dcc7ff6d9a05d73d1 goes back to `const`, with a `mutable` mutex. Thanks @dirk-thomas for the tip!